perfkitbenchmarker/scripts/database_scripts/launch_driver.sh

```diff
@@ -31,20 +30,24 @@
 # Example call:
 #    ./perfkitbenchmarker/scripts/database_scripts/launch_driver.sh bucket_name
 
-# To utilize per second data graphs, include a storage bucket as a second input.
+# To utilize per second data graphs, indicate that with a second input.
 # Example:
-#     ./perfkitbenchmarker/scripts/database_scripts/launch_driver.sh log_bucket_name graph_bucket_name
+#     ./perfkitbenchmarker/scripts/database_scripts/launch_driver.sh log_bucket_name per_second_graphs
 
 
 if [ -z $1 ]; then
   echo "Must pass in a cloud storage bucket."
   exit 1;
 fi
 
+cloud_storage_bucket=$1
+per_second_graph=$2
+
+
 while true
   do
   # provision, prepare phase of mysql_service
-  run_uri=$(python perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py --sysbench_run_seconds="1200" --run_stage=provision,prepare --mysql_svc_db_instance_cores="16" --mysql_svc_oltp_table_size="12000000" --mysql_svc_oltp_tables_count="100" --mysql_instance_storage_size=1000 --additional_flags='"'"--cloud_storage_bucket=${1}"'"')
+  run_uri=$(python perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py --run_stage=provision,prepare --mysql_svc_db_instance_cores="16" --mysql_svc_oltp_table_size="12000000" --mysql_svc_oltp_tables_count="100" --mysql_instance_storage_size=1000 --additional_flags='"'"--cloud_storage_bucket=${cloud_storage_bucket}"'"')
```

Comment:
As mentioned in the last round of review, please pull these hard-coded constants out and define them at the start of the bash script, it's easier to manage that way. e.g., "thread_count_list" is referred to twice in the code below, would be much better if there is only one definition of it as a constant variable.